kstreams/middleware/middleware.py

```diff
@@ -61,12 +62,18 @@ async def __call__(self, cr: ConsumerRecord) -> typing.Any:
 
 
 class ExceptionMiddleware(BaseMiddleware):
```

Comment:
For future reference, it would be nice to have a docstring explaining a bit more here. Some things that would be nice to know:

- how it works
- if the user is expected to use this middleware or not
- if it's possible to extend it

Reply:
It is not possible to use this `Middleware`, this is why there is not documentation